src/test/java/org/fundacionjala/pivotal/steps/api/APIProjectSteps.java

```diff
@@ -23,7 +37,9 @@
      */
     @And("^I post a new \"([^\"]*)\" to \"([^\"]*)\" endpoint$")
     public void iPostANewToEndpoint(final String feature, final String endpoint, final Map<String, String> values) {
+        System.out.println("test");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)